examples/3dsphere/baroclinic_wave_ρθ.jl

```diff
@@ -0,0 +1,51 @@
+using ClimaCorePlots, Plots
+
+include("baroclinic_wave_utilities.jl")
+
+driver_values(FT) = (;
+    zmax = FT(30.0e3),
+    velem = 10,
+    helem = 4,
+    npoly = 4,
+    tmax = FT(60 * 60 * 24 * 10),
+    dt = FT(400.0),
+    ode_algorithm = OrdinaryDiffEq.Rosenbrock23,
+    jacobian_flags = (; ∂𝔼ₜ∂𝕄_mode = :exact, ∂𝕄ₜ∂ρ_mode = :exact),
+    max_newton_iters = 2,
+    save_every_n_steps = 10,
+    additional_solver_kwargs = (;), # e.g., reltol, abstol, etc.
+)
+
+initial_condition(local_geometry) = initial_condition_ρθ(local_geometry)
+initial_condition_velocity(local_geometry) =
+    initial_condition_velocity(local_geometry; is_balanced_flow = false)
+
+remaining_cache_values(Y, dt) = merge(
+    baroclinic_wave_cache_values(Y, dt),
+    final_adjustments_cache_values(Y, dt; use_rayleigh_sponge = false),
+)
+
+function remaining_tendency!(dY, Y, p, t)
+    fill!(dY, zero(eltype(dY)))
```

Comment:
```suggestion
    dY .= 0
```
This seems to be allocating ~341 MiB at every `step!`.